scripts/lib/upgrade-zulip-stage-2

```diff
@@ -125,9 +125,17 @@ elif args.from_git:
     # update-prod-static with the git upgrade process.  But it'll fail
     # safely; this seems like a worthwhile tradeoff to minimize downtime.
     logging.info("Building static assets...")
-    subprocess.check_call(["./tools/update-prod-static", "--authors-not-required", "--prev-deploy",
-                           os.path.join(DEPLOYMENTS_DIR, 'current')],
-                          preexec_fn=su_to_zulip)
+    try:
+        subprocess.check_call(["./tools/update-prod-static", "--authors-not-required", "--prev-deploy",
+                               os.path.join(DEPLOYMENTS_DIR, 'current')],
+                              preexec_fn=su_to_zulip)
+    except subprocess.CalledProcessError:
+        # Print proper error output in-case update-prod-static fails.
+        log_file_path = "/home/zulip/deployments/next/var/log/update-prod-static.log"
+        logging.error("Building static assets failed.")
+        logging.info("Complete error message can be fount at " + log_file_path)
```

Comment:
typo: fount -> found

Reply:
It's not important for this commit, but something to keep in mind for future cleanup--we generally use the `run` helper from `scripts/lib/zulip_tools.py` as a wrapper for `subprocess.check_call`, and it catches ` subprocess.CalledProcessError`, gives a little extra info, then re-raises the exception.  I'm not sure that's the best thing here--you are trying to reduce the noise in the exception, apparently, which can be helpful at time--but I wonder if we want another helper for these types of scenarios.

Even though the wrappers are pretty thing wrappers, I do think it's helpful to have consistent error handling.

Reply:
Sorry for the typo. I agree the whole error handling approach can be done better.